langchain4j-azure-ai-search/src/test/java/dev/langchain4j/store/embedding/azure/search/AzureAiSearchEmbeddingStoreIT.java

```diff
@@ -0,0 +1,121 @@
+import dev.langchain4j.model.embedding.EmbeddingModel;
+import dev.langchain4j.store.embedding.*;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.condition.EnabledIfEnvironmentVariable;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.List;
+
+import static dev.langchain4j.internal.Utils.randomUUID;
+import static java.util.Arrays.asList;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.data.Percentage.withPercentage;
+
+@EnabledIfEnvironmentVariable(named = "AZURE_SEARCH_ENDPOINT", matches = ".+")
+public class AzureAiSearchEmbeddingStoreIT extends EmbeddingStoreIT {
+
+    private static final Logger log = LoggerFactory.getLogger(AzureAiSearchEmbeddingStoreIT.class);
+
+    private EmbeddingModel embeddingModel;
+
+    private EmbeddingStore<TextSegment> embeddingStore;
+
+    public AzureAiSearchEmbeddingStoreIT() {
+
+        embeddingModel = new AllMiniLmL6V2QuantizedEmbeddingModel();
+
+        embeddingStore =  AzureAiSearchEmbeddingStore.builder()
+                .endpoint(System.getenv("AZURE_SEARCH_ENDPOINT"))
+                .apiKey(System.getenv("AZURE_SEARCH_KEY"))
+                .embeddingModel(embeddingModel)
+                .build();
+    }
+
+    @Test
+    void testAddEmbeddingsAndFindRelevant() {
+        String content1 = "banana";
+        String content2 = "computer";
+        String content3 = "apple";
+        String content4 = "pizza";
+        String content5 = "strawberry";
+        String content6 = "chess";
+        List<String> contents = asList(content1, content2, content3, content4, content5, content6);
+
+        for (String content : contents) {
+            TextSegment textSegment = TextSegment.from(content);
+            Embedding embedding = embeddingModel.embed(content).content();
+            embeddingStore.add(embedding, textSegment);
+        }
+
+        awaitUntilPersisted();
+
+        Embedding relevantEmbedding = embeddingModel.embed("fruit").content();
+        List<EmbeddingMatch<TextSegment>> relevant = embeddingStore.findRelevant(relevantEmbedding, 3);
+        assertThat(relevant).hasSize(3);
+        assertThat(relevant.get(0).embedding()).isNotNull();
+        assertThat(relevant.get(0).embedded().text()).isIn(content1, content3, content5);
+        log.info("#1 relevant item: {}", relevant.get(0).embedded().text());
+        assertThat(relevant.get(1).embedding()).isNotNull();
+        assertThat(relevant.get(1).embedded().text()).isIn(content1, content3, content5);
+        log.info("#2 relevant item: {}", relevant.get(1).embedded().text());
+        assertThat(relevant.get(2).embedding()).isNotNull();
+        assertThat(relevant.get(2).embedded().text()).isIn(content1, content3, content5);
+        log.info("#3 relevant item: {}", relevant.get(2).embedded().text());
+    }
+
+    @Test
+    void should_return_correct_score() {
```

Comment:
this test can probably be removed as it is no different from original method except it adds logging. you can add logging to the base method if you want